build/build.sh

```diff
@@ -62,3 +62,5 @@ cd "${PROJECTDIR}"
 "${SCRIPTDIR}/build-libbsd.sh" clean install
 "${SCRIPTDIR}/build-libinih.sh"
 "${SCRIPTDIR}/build-libgrisp.sh"
+
+git log -1 --format="%H" > buildrev
```

Comment:
We should use `git rev-parse HEAD` here instead of `git log`. `git log` isn't made for programmatic access like this.